stable/atlantis/README.md

```diff
@@ -32,6 +32,7 @@ The following options are supported.  See [values.yaml](values.yaml) for more de
 | ------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------- |
 | `allow_repo_config` | Whether to allow the use of [atlantis.yaml files](https://www.runatlantis.io/docs/atlantis-yaml-reference.html). | `false`                  |
 | `atlantis_data_storage`                     | The amount of storage available for Atlantis' data directory (mostly used to check out git repositories).                                                                                                                                                                                                                                                  | `5Gi`                  |
+| `atlantis_data_storageClass`                     | Storage class of the volume mounted for the Atlantis data directory.                                                                                                                                                                                                                                                  | n/a                  |
```

Comment:
shouldn't these all be camelCase?

Reply:
Also why is it prefixed with `atlantis_`? I don't think that's necessary given this is the Atlantis chart.